### PR TITLE
HW1

### PR DESCRIPTION
**Pull request by VladykaIhor**

### CODE REVIEW

File: Main.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package HomeWork1;
+
+public class Main {
```
Comment (Sofasmile) [inline Main.java]: better use Application than Main

File: Main.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
+package HomeWork1;
```
Comment (Sofasmile) [inline Main.java]: try not use numbers in the name of the package HomeWork1 -> firstHomework
https://mate-academy.github.io/style-guides/java/java.html#s5-naming

File: MatchResolution.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
+package HomeWork1;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class MatchResolution {
+    protected static int teamOneScore, teamTwoScore, teamOneBet, teamTwoBet;
+
+    public static void main(String[] args) throws IOException {
```
Comment (Sofasmile) [inline MatchResolution.java]: why do you need this method here?

File: MatchResolution.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+    }
+
+    protected static int calculateWinIndex(int teamOneScore, int teamTwoScore, int teamOneBet, int teamTwoBet) {
+        return teamOneScore == teamOneBet & teamTwoScore == teamTwoBet ? 2 :
```
Comment (Sofasmile) [inline MatchResolution.java]: use && than &

File: MatchResolution.java [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+                teamOneScore == teamTwoScore &&
+                        teamOneBet == teamTwoBet || teamOneScore > teamTwoScore && teamOneBet > teamTwoBet ||
+                        teamOneScore < teamTwoScore && teamOneBet < teamTwoBet ? 1 :
+                        0;
```
Comment (Sofasmile) [inline MatchResolution.java]: better move it into previous line

File: MathPower.java [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class MathPower {
+    public static int number;
+    public static int customPowerNumber;
+
+    public static void main(String[] args) {
```
Comment (Sofasmile) [inline MathPower.java]: the same

File: MathPower.java [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+    }
+
+    public static int powerOfTwo(int number) {
+        return number * number;
+    }
+
```
Comment (Sofasmile) [inline MathPower.java]: remove one redundant line

File: Rhombus.java [hunk 8]
```diff
@@ -0,0 +1,29 @@
+package HomeWork1;
+
+public class Rhombus {
+/*As far as I know HomeWork1.Rhombus is a geometrical figure,
```
Comment (Sofasmile) [inline Rhombus.java]: it was optional comment.
Next time do not write comments in the code.
Write comments when creating a PR

File: Rhombus.java [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+            for (int m = 0; m <= rhombusMiddleLineSize - i; m++) {
+                System.out.print(" ");
+            }
+            for (int j = 0; j <= i; j++) {
+                System.out.print("* ");
+            }
+            System.out.println();
+        }
+    }
+}
```
Comment (Sofasmile) [inline Rhombus.java]: add a blank line after }

File: Sequences.java [hunk 10]
```diff
@@ -0,0 +1,33 @@
+package HomeWork1;
+
+public class Sequences {
+    static int n1 = 0, n2 = 1, n3 = 0;
```
Comment (Sofasmile) [inline Sequences.java]: create variables in the separate lines

File: Sequences.java [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+        }
+    }
+    static int factorial(int num) {
+        int result = 1;
+        if (num >= 1) {
+            result = (factorial(num - 1)) * num;
+        } else result = 1;
+        return result;
+    }
+
```
Comment (Sofasmile) [inline Sequences.java]: remove redundant line

File: Sequences.java [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+            n2 = n3;
+            System.out.print(" " + n3);
+            fibonacci(count - 1);
+        }
+    }
+    static int factorial(int num) {
+        int result = 1;
+        if (num >= 1) {
+            result = (factorial(num - 1)) * num;
+        } else result = 1;
```
Comment (Sofasmile) [inline Sequences.java]: use full else, if, for statements
else {
result = 1;
}

File: Main.java (hunk 1 above)
Comment (VladykaIhor) [reply Main.java]: Okay

File: MatchResolution.java (hunk 3 above)
Comment (VladykaIhor) [reply MatchResolution.java]: Just forgot to remove it after cleaning it up. Fixin'it)

File: Rhombus.java (hunk 8 above)
Comment (VladykaIhor) [reply Rhombus.java]: Okay

**Pull request closed without merging**